src/vs/workbench/contrib/list/browser/listResizeColumnAction.ts

```diff
@@ -0,0 +1,41 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { TableColumnResizeQuickPick } from 'vs/workbench/contrib/list/browser/tableColumnResizeQuickPick';
+import { Table } from 'vs/base/browser/ui/table/tableWidget';
+import { KeyCode } from 'vs/base/common/keyCodes';
+import { IInstantiationService, ServicesAccessor } from 'vs/platform/instantiation/common/instantiation';
+import { KeybindingWeight } from 'vs/platform/keybinding/common/keybindingsRegistry';
+import { IListService, WorkbenchListFocusContextKey } from 'vs/platform/list/browser/listService';
+import { Action2 } from 'vs/platform/actions/common/actions';
+import { localize } from 'vs/nls';
+
+export class ListResizeColumnAction extends Action2 {
+	constructor() {
+		super({
+			id: 'list.resizeColumn',
+			title: { value: localize('list.resizeColumn', "Resize Column"), original: 'Resize Column' },
+			category: { value: localize('list', "List"), original: 'List' },
+			keybinding: {
+				primary: KeyCode.F8,
```

Comment:
F8 is one of my very commonly used keybindings, always annoys me when extensions override this.

Reply:
Oh yes, I had forgotten to remove this keybinding, thanks